README.md

```diff
@@ -7,13 +7,10 @@
 
 ## Overview
 
-Sonobuoy addresses the gap in tools for [Kubernetes][3] cluster introspection. It is a customizable, extendable, and cluster-agnostic way to record snapshots of your cluster's essential characteristics.
-
-Sonobuoy generates clear, informative reports about your cluster, regardless of your deployment details. Its selective data dumps of Kubernetes resource objects and cluster nodes allow for the following use cases:
+Sonobuoy addresses the gap in tools for [Kubernetes][3] cluster introspection. It is a customizable, extendable, and cluster-agnostic way to generate clear, informative reports about your cluster--regardless of your deployment details. Its selective data dumps of Kubernetes resource objects and cluster nodes allow for the following use cases:
```

Comment:
^ @timothysc  do you mean that I should instead phrase this as "*It's a selective data dump* of Kubernetes resource objects and cluster nodes *, which allow* for the following use cases"?